packages/sensors_plus/LICENSE

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Here are my review comments for battery_plus_tizen:

- https://github.com/flutter-tizen/plugins/pull/120#discussion_r644451282
- https://github.com/flutter-tizen/plugins/pull/120#discussion_r644451413

Reply:
I missed your review! thank you~